1.0/tools/optimize-for-production.md

```diff
@@ -200,7 +200,9 @@ This results in a `elements.vulcanized.html` that looks a little like this:
 
 [Content Security Policy](http://en.wikipedia.org/wiki/Content_Security_Policy) (CSP) is a JavaScript security model that aims to prevent XSS and other attacks. In so doing, it prohibits the use of inline scripts.
 
-To use {{site.project_title}} in a CSP environment that doesn't support inline scripts, you can use the Crisper project. Crisper removes all scripts from the HTML Imports and places their contents into an output JavaScript file. This is useful in amongst other things, using {{site.project_title}} in a Chrome App.
+The Polymer build tools can also be used to transform your code to run in an environment that enforces content security policy (CSP), including Chrome Apps and Extensions.
```

Comment:
Can we move this paragraph right up to the top (like, L14), so the intro to this doc explicitly mentions CSP (maybe with a link to this section)?